esmvalcore/preprocessor/_time.py

```diff
@@ -1559,7 +1559,9 @@ def _transform_to_lst_eager(
     """
     # Apart from the time index, all other dimensions will stay the same; this
     # is ensured with np.ogrid
-    idx = np.ogrid[tuple(slice(0, d) for d in data.shape)]
+    # NOTE: in numpy 2.0 np.ogrid returns ndarray or tuple of ndarrays
```

Comment:
Remove comment? This should be fully backwards-compatible, so I don't think it's useful.

Reply:
done in 05dea6644aee29f5da34d64215c71cbb3e6920a9